libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -9,16 +9,17 @@ import {
 } from '@tmo/books/data-access';
 import { FormBuilder } from '@angular/forms';
 import { Book } from '@tmo/shared/models';
-import { Observable } from "rxjs";
+import { Subject, Observable} from "rxjs";
+import {debounceTime, distinctUntilChanged, takeUntil } from "rxjs/operators";
 
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
   styleUrls: ['./book-search.component.scss']
 })
-export class BookSearchComponent {
-  books: ReadingListBook[];
+export class BookSearchComponent implements OnInit, OnDestroy {
   books$: Observable<ReadingListBook[]>;
+  unsubscribe$: Subject<any> = new Subject<any>();
```

Comment:
You could just use `unsubscribe$: Subject<void> = new Subject();` instead. This would make the call on line 74 just `this.unsubscribe$.next();`

Reply:
Updated.